Keyboards/KeyboardsBase/KeyboardStyling.swift

```diff
@@ -30,6 +30,21 @@ func styleBtn(btn: UIButton, title: String, radius: CGFloat) {
   btn.layer.masksToBounds = false
   btn.layer.cornerRadius = radius
   btn.setTitle(title, for: .normal)
+  // Displays info button for missing data
```

Comment:
Can you do a comma at the end of inline comments? Kind of a pet nitpick for me, but I learned from Python styleguides that comments on a line should be a full sentence including capitalization and a period, and inline should be lower case and not have a period :)